symplyphysics/laws/electricity/circuits/approximation_of_transmission_coefficient_of_normalized_lpf.py

```diff
@@ -0,0 +1,60 @@
+from sympy import (
+    Expr,
+    Eq,
+    solve,
+)
+from symplyphysics import (
+    SI,
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+    convert_to_float
+)
+
+# Description
+## The approximation of the power transmission coefficient of a normalized low-pass filter is given by approximating functions of the order of n.
+## Bandwidth distortion determines the maximum distortion in the bandwidth. In other words, to determine the level of ripples in the bandwidth.
+
+# Law: H = 1 / (1 + e^2 * Fn(w)^2)
+## Where:
+## H - the transfer coefficient of the filter,
+## e - bandwidth distortion,
+## Fn(w) - approximating function of the order of n,
+## w - frequency normalized to the cutoff frequency(the frequency that corresponds to the maximum bandwidth distortion).
```

Comment:
Does it matter if it is normalized or not? Function could apply normalization itself, so any frequency should suffice.